vpr/src/place/place.cpp

```diff
@@ -448,8 +468,92 @@ void try_place(const t_placer_opts& placer_opts,
         }
     }
 
-    move_generator = std::make_unique<UniformMoveGenerator>();
+    int move_lim = 1;
```

Comment:
This is quite a bit of code to add to an already long routine.  I think it would be better if you put it all in a helper function or two (something like init_place_move_types () but maybe there's a better name).

Reply:
Fixed by adding a new helper function "create_move_generators()" that returns two unique pointers to the two move generators.